src/components/info-panel/InfoPanel.tsx

```diff
@@ -133,7 +133,7 @@ export function InfoPanel(props: InfoPanelProps) {
 function valueCell(value: number | string | object | boolean) {
   switch (typeof value) {
     case 'boolean':
-      return <ValueRenderers.Boolean value={value} />;
+      return <ValueRenderers.Boolean renderValue={String} value={value} />;
```

Comment:
Why not just calling a Text renderer with String(value) ? I don't see why we should change the Boolean renderer.

Reply:
we did this to render the value as a string "true" or "false" instead of a symbol

Reply:
@lpatiny's suggestion would give the same result.

Reply:
The purpose is to provide users of the ValueRenderers.Boolean component with the flexibility to customize the rendering of strings, allowing them to choose representations that best convey true or false values according to their preferences